modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,116 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies implement an additional level of granularity with SLA tiering. Policies control access based on the following access structuring:
+
+* Resource-level policies
+Impose a limit on all requests that match the resource to which it is applied. All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+* SLA-based policies
+Impose a limit specified by the level of access granted to the requesting application. The Rate Limiting policy is SLA-based.
+
+When the API Gateway is configured to work as a Mule Cluster, the policy can be configured to work in a distributed manner.
+  
+  [PA: This seems out of place and does not make sense.]
+
+[PA: We need to tell the customers how to work with SLA-tiers for policies right?]
+
+== Resource-Level Policies
+
+Resource level policy support includes RAML and, since Mule 4, HTTP-based APIs. A policy applied at a resource level affects all HTTP methods (eg PUT, GET, POST), or selected HTTP methods, within the resource.
+
+image::methods-resources.png[]
+
+You can apply multiple conditions to filter your resources and HTTP methods. The URI template regex can be applied to one, many or all the methods that your API has.
+
+IMPORTANT: You cannot apply the CORS policy to resources.
+
+=== Usage Scenarios
+
+The uses for resource level policies are limited mainly by your imagination. Here are a few possibilities:
+
+* Applying policies to specific resources
+* Securing a subset of an API
+* Setting different limits on resources
+
+If you have a RAML/OAS spec attached to your API, click Preview Resource Matching to check which resources your filters affect.
+
+=== Concepts
+
+Example API:
+
+image::users-api-cropped.png[API Console displays resources and methods,height=329,width=732]
+
+A resource level policy supports Java regular expressions. For example, you can use the wildcard to apply a policy to multiple resources. When you apply the policy to the API, specify the resources to which it applies.
```

Comment:
@balbifm were there differences between mule 3 and mule 4 in the expressions? if there are we should highlight them here

Reply:
Will do! Thanks!